### PR TITLE
Django 1.9

### PR DESCRIPTION
**Pull request by janno42**
update to django 1.9

### CODE REVIEW

File: _1327/urls.py [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+from _1327.user_management import views as user_management_views
+
 admin.autodiscover()
 
-urlpatterns = patterns('_1327',  # noqa
-	url(r"^$", 'main.views.index', name='index'),
+urlpatterns = [
+	url(r"^$", main_views.index, name='index'),
 	url(r"^documents/", include('_1327.documents.urls', namespace='documents')),
 	url(r"^minutes/", include('_1327.minutes.urls', namespace='minutes')),
```
Comment (invliD) [inline _1327/urls.py]: What about also importing these and referencing the actual urls module instead of a string here?

Comment (janno42) [inline _1327/urls.py]: i think this is the preferred way, at least according to the docs (https://docs.djangoproject.com/en/1.9/topics/http/urls/#including-other-urlconfs)

Comment (invliD) [inline _1327/urls.py]: I don't see any comment about which form is preferred, but it's fine either way, I guess.

**Comment by invliD** [comment]
:+1:

**Pull request merged**